src/nvim/debugger.c

```diff
@@ -50,7 +50,7 @@ struct debuggy {
 void do_debug(char *cmd)
 {
   int save_msg_scroll = msg_scroll;
-  int save_State = State;
+  int save_state = State;
```

Comment:
this only makes sense togheter with changing `State` to `state` (or something else, to make in more unique as a global var)

Reply:
we could skip this rule for now.